homeassistant/components/sensor/waze_travel_time.py

```diff
@@ -94,6 +99,7 @@ def __init__(self, name, origin, destination, region,
         self._state = None
         self._origin_entity_id = None
         self._destination_entity_id = None
+        self.update = Throttle(interval)(self.update)
```

Comment:
Just add the throttle as a decorator to `update` with fixed throttling time.

Reply:
Thank you for the feedback, I was Fix the interval 
Can you clarify a little more about this one?

Reply:
We should set the minimum interval between updates to a safe fixed time interval. That doesn't need to be configurable.
```py
@Throttle(MIN_TIME_BETWEEN_UPDATES)
def update(self):
   ...
```

Note that a throttle on the entity update won't hinder other entities from making requests to the api within this minimum time interval. We're not throttling the whole platform just per entity. Is this ok?

Reply:
Excellent explanation, updated the code, thank you